mountinfo/mountinfo_linux.go

```diff
@@ -194,7 +194,7 @@ func unescape(path string) (string, error) {
 			v := c - '0'
 			for j := 2; j < 4; j++ { // one digit already; two more
 				x := s[j] - '0'
-				if x < 0 || x > 7 {
```

Comment:
It's changed because

```
mountinfo_linux.go:197:8: SA4003: no value of type byte is less than 0 (staticcheck)
```

Reply:
Do you mind creating a PR adding this linter (or linters) to this repo CI?

Reply:
Oh OK nevermind, it's there already but the non-zero exit code is currently ignored by make.